### ISSUE TITLE
[Feature Request]Add table of contents to each page

### ISSUE THREAD

**Issue by dconman** [emfmesquita/beyondhelp#10]
As far as I can tell, on any given page of a rulebook, to get to a different chapter you must go back to the Table of Contents, then go to the corresponding chapter. It would be nice if somewhere on the page, either at the top or along the side collapsible there were links to the other chapters in the rulebook

**Comment by emfmesquita** [comment]
That is a good idea. I suffer from this myself XD. Added to the list of to do. Thanks.

**Comment by dconman** [comment]
I started working on this in my own [fork ](https://github.com/dconman/beyondhelp) It works, but it definitely could be better designed. I'll work on it some more soon. Any suggestions would be appreciated.

**Comment by emfmesquita** [comment]
That is amazing news! Thanks for the contribution! I will definitely take a look at it.

**Comment by emfmesquita** [comment]
I just tested you PR, and it looks amazing! Thanks again for the contribution. I just found two small things (I will add the comments on PR), and I think we can merge it.

**Comment by emfmesquita** [comment]
I changed my mind, just merged it. I will make the small changes XD. Basically I just added and empty array on TOCData for rules (it was trowing an ex for rule books). And on TOCService I just added a return if the book is not found on TOCData (It was removing the menu on books with no entry on data).

So it will be on the next release. Probably I will do this weekend after I finish the damn extra map refs XD.

Btw feel free to create more PRs if you want to change something.

Thanks again!
